### DISCUSSION TITLE
Integrate "3DYD Youtube Source" to play more video streaming sites

### DISCUSSION THREAD

**Discussion by trimechee** [smplayer-dev/smplayer#205] [Ideas]
Hello, we would like that  SMPlayer integrate "3DYD Youtube Source" to play videos from streaming sites, because I tried to play this video with SMPlayer with 3DYD Youtube Source installed

https://video.sibnet.ru/video2945354-NS_093_VF_www_vostfree_com/?utm_source=player&utm_medium=video&utm_campaign=EMBED

 but SMPlayer failed, so I had to install potplayer and I was able to play the video by putting in 3DYD Youtube Source the yt-dlp option .....
I would like to play this streaming video with SMPlayer because it is less heavy in resources and more fluid and I can increase the network cache but SMPlayer fails to play this video and I hope you SMPlayer can solve the problem and integrate 3DYD natively Youtube Source and maybe integrate DirectShow filters so SMPlayer would become even more beautiful in sha Allah

https://ys.3dyd.com/

**Comment by smplayer-dev** [comment]
I think this is fixed in [version 9953](https://github.com/smplayer-dev/smplayer/releases/tag/v21.8.0.9953). Go to preferences -> network and select yt-dlp.

**Comment by trimechee** [comment]
Great ! Thank you very much for your help :) please i have an other
important request : please add support of DirectShow :

 Hello, i think that our beloved SMPlayer is not compatible with this great
software "3DYD Youtube Source" which is a set of plugins for watching
videos from media sites directly in the video player

https://ys.3dyd.com/home/

because SMPlayer doesn't support playback using DirectShow, so I had to
change my favorite video player SMPlayer for another video player that
supports directshow, please it will be great if there an option to support
directshow in SMPlayer to use this program which allows to avoid using a
web browser that might have been heavy cpu and full of ads !

SMplayer can't play video from sibnet.ru even with video support enabled in
SMPlayer preferences or with 3DYD youtube source" software :

https://video.sibnet.ru/video3492086-Tokubetsuban_Free_Take_Your_Marks_01_VOSTFR_www_vostfree_com/?utm_source=player&utm_medium=video&utm_campaign=EMBED

Thank you !
